src/NzbDrone.Core/Notifications/Stash/StashProxy.cs

```diff
@@ -23,6 +23,19 @@ public void Update(StashSettings settings, string path)
             var generatePreviews = settings.GeneratePreviews ? "true" : "false";
             var cleanPath = path.ToJson();
 
+            var metadataIdentifyQuery = "";
+            if (settings.MetadataIdentify) {
```

Comment:
compile lint error: `error SA1500: Braces for multi-line statements should not share line`

```
if (settings.MetadataIdentify) 
{
```